ks-svarut/src/main/java/no/difi/meldingsutveksling/ks/svarinn/SvarInnMessage.java

```diff
@@ -56,6 +56,9 @@ EDUCore toEduCore() {
         // The payload util doesn't correctly handle payloads that are MeldingType.
         // And I am afraid of fixing that bug might lead to trouble in the archive system.
         String payload = payloadConverter.marshallToString(meldingType).replaceAll(":ns2|ns2:", "");
+        // TODO: Quickfix due to demo 30.05.17 - possible bug in ephorte? See jira issue MOVE-259
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1135)